### PR TITLE
Sendlead

### PR DESCRIPTION
**Pull request by ainsleywagon**
Changing text to say "Congratulations you applied to these programs, the agencies will be in touch in a week."

### CODE REVIEW

**Comment by MrMaksimize** [comment]
Need to resolve merge conflict here first.  I'll still make some comments on code review, but that blocks this from pr going in

File: lib/OppQueryForm.js [hunk 1]
```diff
@@ -34,13 +34,16 @@ function OppQueryForm() {
           choiceOther: true,
           choices: choicesList.getFormChoices('purpose'),
           widget: 'multiSelect'
+          //validate: {
```
Comment (MrMaksimize) [inline lib/OppQueryForm.js]: Commented code should be removed ;)

File: views/leadSentConfirmation.jade [hunk 2]
```diff
@@ -2,10 +2,13 @@ extends layout
 
 block content
   div.congratulations-message
-    h2 Congratulations, #{leadData.name} we sent your info to the agency.
-    h2 An employee will be contacting you soon!
+    h2 Congratulations, #{leadData.name}!
+    h2 You applied for #{selectedPrograms.name}
+
+    - console.log(leadData);
```
Comment (MrMaksimize) [inline views/leadSentConfirmation.jade]: lol think this one might one of my debugs, but needs removing

File: views/leadSentConfirmation.jade [hunk 3]
```diff
@@ -2,10 +2,11 @@ extends layout
 
 block content
   div.congratulations-message
-    h2 Congratulations, #{leadData.name} we sent your info to the agency.
-    h2 An employee will be contacting you soon!
+    h2 Congratulations, #{leadData.name}!
+    h2 You applied for #{selectedPrograms.name}
```
Comment (ainsleywagon) [inline views/leadSentConfirmation.jade]: Hey @MrMaksimize, this doesn't really work right now- it's not grabbing the programs applied for, I was just guessing - so it'd be awesome to look at this together and fixxx.

**Comment by MrMaksimize** [comment]
@ainsleywagon added loop see here https://github.com/CoquiCoders/bizwallet/commit/e22ed36e05864d35885768f8ab0120db8ae01d51

Need some styling love :)  http://take.ms/CB4Gg

**Pull request merged**